src/clic.adoc

```diff
@@ -1586,20 +1590,21 @@ by the additional {tvt} CSR.
 [source]
 ----
  mode submode PC on Interrupt
- 00   xxxx    OBASE                   # CLINT direct mode
- 01   xxxx    OBASE+4*exccode         # CLINT vectored mode
- 11   0000    shv ? M[VTBASE+XLEN/8*exccode)]&~1 : NBASE  # CLIC mode
+ 00   xxxx    OBASE                                             # CLINT direct mode
+ 01   xxxx    OBASE+4*exccode                                   # CLINT vectored mode
+ 11   0000    shv ? (M[VTBASE+XLEN/8*exccode)] & MASK) : NBASE  # CLIC mode
```

Comment:
MASK is not defined

Reply:
MASK and all the others such as OBASE, VTBASE, etc are defined together below the pseudo-code. I don't really think you want me to just define MASK above the pseudo-code. See the whole pseudo-code section below for context. I'm assuming you don't want me to make the requested change after you see this response from me:
`----
 mode submode PC on Interrupt
 00   xxxx    OBASE                                             # CLINT direct mode
 01   xxxx    OBASE+4*exccode                                   # CLINT vectored mode
 11   0000    shv ? (M[VTBASE+XLEN/8*exccode)] & MASK) : NBASE  # CLIC mode
 10   0000    Reserved
 1x   yyyy    Reserved

where:
  OBASE  = xtvec[XLEN-1:2]<<2   # CLINT mode vector base is at least 4-byte aligned
  NBASE  = xtvec[XLEN-1:6]<<6   # CLIC mode vector base is at least 64-byte aligned
  VTBASE = xtvt[XLEN-1:6]<<6    # Vector table base is at least 64-byte aligned
  shv    = clicintrattr[i].shv
  x      = any value (don't care)
  yyyy   = any non-zero value
  M[a]   = Contents of memory address at address "a"
  MASK   = ~0x1 if IALIGN=16 or ~0x3 if IALIGN=32`